security-and-privacy-questionnaire-unrestricted-webusb.md

```diff
@@ -0,0 +1,125 @@
+# [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
```

Comment:
Thank you for including notes on where WebUSB and Unrestricted WebUSB differ. I think we can rename this file to `security-and-privacy-questionnaire.md` since it adiquately covers the original API as well.

Reply:
Renamed